src/context_store/storage/prisma.py

```
@@ -265,172 +375,96 @@ def __init__(self, client: Prisma) -> None:
-                version,
-            )
-
-    async def _tables_exist(self, table_names: list[str]) -> bool:
-        rows = await self._client.query_raw(
-            "SELECT tablename FROM pg_catalog.pg_tables WHERE tablename = ANY($1::text[])",
-            table_names,
-        )
-        return len(rows) == len(table_names)
+            rows = await self._client.query_raw(sql)
+            return {row["version"] for row in rows}
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)